conan/tools/cmake/toolchain/blocks.py

```diff
@@ -552,6 +561,7 @@ def context(self):
             "cmake_include_path": self._join_paths(host_include_paths),
             "is_apple": is_apple_,
             "cross_building": cross_building(self._conanfile),
+            "lib_dirs": self._join_paths(host_lib_dirs)
```

Comment:
Maybe better call it ``host_lib_dirs``? Actually, if this is intended for the RUNTIME, ``host_runtime_libdirs`` or something in that line would be more explicit and clear.